gnovm/tests/file.go

```diff
@@ -632,6 +634,21 @@ func trimTrailingSpaces(result string) string {
 	return strings.Join(lines, "\n")
 }
 
+// ----------------------------------------
+// testParams
+type testParams struct{}
+
+func newTestParams() *testParams {
+	return &testParams{}
+}
+
+// XXX: not noop?
```

Comment:
Noop is good, much better than random setters executing without you even knowing within a test context 👍

Reply:
23f8ce2ac